impeller/entity/contents/content_context.h

```diff
@@ -437,6 +506,83 @@ class ContentContext {
       ContentContextOptions opts) const {
     return GetPipeline(blend_softlight_pipelines_, opts);
   }
+#if FML_OS_PHYSICAL_IOS
```

Comment:
So reading from the framebuffer wouldn't happen to work on Apple silicon macs, would it? :)

Reply:
it _might_ be, but from the errors I hit it sounds like it needs a substantially higher MSL version:

https://github.com/KhronosGroup/SPIRV-Cross/blob/4e2fdb25671c742a9fbe93a6034eb1542244c7e1/spirv_msl.cpp#L13082